lib/suse-cloud-upgrade-4-to-5-pre

```diff
@@ -205,6 +205,21 @@ for node in $allocated_nodes; do
   ssh "$node" 'for i in /etc/init.d/openstack-* /etc/init.d/openvswitch-switch /etc/init.d/ovs-usurp-config-* /etc/init.d/drbd /etc/init.d/openais; do if test -e $i ; then initscript=`basename $i`; chkconfig -d $initscript; $i stop; fi; done'
 done
 
+
+### Workaround for neutron shortcoming
+# Currently neutron-ns-metadata-proxy is not stoppend on a HA setup. The running process
+# kills the update, so we need to kill it manually. Upstream is working on improving the
+# handling of external processes. Once this is achieved we don't need this part anymore
```

Comment:
Indeed there's a blueprint approved for that [1] .
Anyway the blueprint doesn't take care of killing orphaned external processes. This can be proposed upstream. So I guess that for now we have to do it manually.

[1] https://blueprints.launchpad.net/neutron/+spec/agent-child-processes-status